<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MoBookDialog</class>
 <widget class="QDialog" name="MoBookDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QToolButton" name="backBtn">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>50</y>
     <width>30</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string>...</string>
   </property>
  </widget>
  <widget class="QComboBox" name="bookComboBox">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>50</y>
     <width>160</width>
     <height>30</height>
    </rect>
   </property>
  </widget>
  <widget class="QListView" name="wordListView">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>90</y>
     <width>261</width>
     <height>191</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="loadBtn">
   <property name="geometry">
    <rect>
     <x>280</x>
     <y>50</y>
     <width>100</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string>加入规划</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
